packages/config/config/devices/0x5254/zxt-310.json

```diff
@@ -9,44 +9,218 @@
-			"valueSize": 4,
+		"21": {
+			"label": "Download Code Learning",
+			"description": "Enter the command number or key to be downloaded to the device",
+			"valueSize": 2,
 			"minValue": 0,
-			"maxValue": 0,
+			"maxValue": 65280,
 			"defaultValue": 0,
+			"unsigned": true,
 			"readOnly": false,
 			"writeOnly": false,
 			"allowManualEntry": true
 		},
-		"21": {
-			"label": "The Simple AV command",
-			"description": "The Simple AV command number or key going to be downloaded to the ZXT‐310",
+		"22": {
+			"label": "Download Status Register",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 4,
+			"defaultValue": 1,
+			"unsigned": true,
+			"readOnly": true,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Idle",
+					"value": 0
+				},
+				{
+					"label": "Ok",
+					"value": 1
+				},
+				{
+					"label": "Learning",
+					"value": 2
+				},
+				{
+					"label": "Command full",
+					"value": 3
+				},
+				{
+					"label": "Learning fail",
+					"value": 4
+				}
+			]
+		},
+		"25": {
+			"label": "IR Code Learning",
+			"description": "Enter the command number or key to be learned. Hold your remote 1-3 cm from the 'L' mark on the device and hold the button to be learned for several seconds",
 			"valueSize": 2,
 			"minValue": 0,
-			"maxValue": 0,
+			"maxValue": 65280,
 			"defaultValue": 0,
+			"unsigned": true,
 			"readOnly": false,
 			"writeOnly": false,
 			"allowManualEntry": true
 		},
-		"22": {
-			"label": "Download status register",
+		"26": {
+			"label": "Learning Status Register",
 			"valueSize": 1,
 			"minValue": 0,
 			"maxValue": 4,
-			"defaultValue": 0,
+			"defaultValue": 1,
+			"unsigned": true,
 			"readOnly": true,
 			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Idle",
+					"value": 0
+				},
+				{
+					"label": "Ok",
+					"value": 1
+				},
+				{
+					"label": "Learning",
+					"value": 2
+				},
+				{
+					"label": "Command full",
+					"value": 3
+				},
+				{
+					"label": "Learning fail",
+					"value": 4
+				}
+			]
+		},
+		"28": {
+			"label": "IR Output Power",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 255,
+			"defaultValue": 255,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
 			"allowManualEntry": true
+		},
+		"29": {
+			"label": "IR Port Mapping",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 6,
+			"defaultValue": 1,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "IR Port 1",
+					"value": 1
+				},
+				{
+					"label": "IR Port 2",
+					"value": 2
+				},
+				{
+					"label": "IR Port 3",
+					"value": 3
+				},
+				{
+					"label": "IR Port 4",
+					"value": 4
+				},
+				{
+					"label": "IR Port 5",
+					"value": 5
+				},
+				{
+					"label": "IR Port 6",
+					"value": 6
+				}
+			]
```

Comment:
Does this need options?

Reply:
I debated that. I'm fine either way but it seems like something you'd pick in a drop down?

Reply:
Ok for me